crates/conjure_core/src/parse/parse_model.rs

```diff
@@ -825,13 +825,13 @@ fn parse_indexing_slicing_op(
         match &mut target {
             Expression::UnsafeIndex(_, new_target, new_indices) => {
                 indices.extend(new_indices.iter().cloned().rev().map(Some));
-                target = *new_target.clone();
```

Comment:
This could be in a later PR (as I guess you did a bulk search+replace), but I'm guessing we don't need this clone() here?

Reply:
`new_target` is a `&mut`  iirc. Maybe a `std::mem::replace` would work here?